aiven_mysql_migrate/utils.py

```diff
@@ -49,13 +62,26 @@ def from_uri(uri: str, name: Optional[str] = None):
         port = res.port or DEFAULT_MYSQL_PORT
         options = parse_qs(res.query)
         ssl = not (options and options.get("ssl-mode", ["DISABLE"]) == ["DISABLE"])
+
+        LOGGER.debug("from_uri - sslca:[%s], sslcert:[%s], sslkey:[%s]", sslca, sslcert, sslkey)
         return MySQLConnectionInfo(
-            hostname=res.hostname, port=port, username=res.username, password=res.password, ssl=ssl, name=name
-        )
+            hostname=res.hostname,
+            port=port,
+            username=res.username,
+            password=res.password,
+            ssl=ssl,
+            sslca=sslca,
+            sslcert=sslcert,
+            sslkey=sslkey,
+            name=name
+            )
 
     def to_uri(self):
         ssl_mode = "DISABLE" if not self.ssl else "REQUIRE"
-        return f"mysql://{self.username}:{self.password}@{self.hostname}:{self.port}/?ssl-mode={ssl_mode}"
+        ssl_auth = urllib.parse.urlencode(f"&ssl-ca={self.sslca}&ssl-cert={self.sslcert}&ssl-key={self.sslkey}") \
```

Comment:
I don't think this will work properly with paths (this should fail `urllib.parse.urlencode(f"&ssl-ca=/a/b/c"`), need to be checked.
But in general the easier way would be to put all the parameters into dict and let urllib build the full parameters str, e.g.
```python
  params = {
    'ssl-mode': ssl_mode,
    'ssl-ca': self.ssl_ca,
    ...
  }
  params_qs = urllib.parse.urlencode(params)
  return f"mysql://{self.username}:{self.password}@{self.hostname}:{self.port}/?{params_qs}"
```
Another thing, why do we need to log those parameters in so many places? Seems a bit unnecessary.